### PR TITLE
config.js => config.ts

### PR DESCRIPTION
**Pull request by jtakalai**
makes adding more Ethereum/DU utils easier, makes tests DRYer

### CODE REVIEW

File: test/integration/config.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import { Wallet } from 'ethers'
```
Comment (teogeb) [inline test/integration/config.ts:1]: Do the benchmark tests and other test written in JS work? 

At least `npm run benchmarks` command fails as `publish.js` JS-file imports to a `config.ts` TypeScript file. Maybe we need to convert all tests to TypeScript, or import the built JS file from `dist` folder.

**Comment by teogeb** [comment]
Some tests have not yet updated? `npm run eslint` fails with errors (mostly `'clientOptions' is not defined`)

File: test/integration/config.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+// can mint mainnet DATA tokens
+export const tokenAdminPrivateKey = '0x5e98cce00cff5dea6b454889f359a4ec06b9fa6b88e9d69b86de8e1c81887da0'
+
+export const providerSidechain = new JsonRpcProvider(clientOptions.sidechain)
+export const providerMainnet = new JsonRpcProvider(clientOptions.mainnet)
+
+export function getTestWallet(index: number, provider: JsonRpcProvider) {
+    // TODO: change to 'streamr-client-javascript' once https://github.com/streamr-dev/smart-contracts-init/pull/36 is in docker
+    const hash = id('marketplace-contracts' + (index || ''))
```
Comment (teogeb) [inline test/integration/config.ts:51]: Is the `|| ''` operation needed? As index is a non-null number that special is applied only if index is zero.

File: test/integration/config.ts (hunk 1 above)
Comment (jtakalai) [reply test/integration/config.ts:1]: I'm unable to run any tests because of babel errors

Comment (jtakalai) [reply test/integration/config.ts:1]: but that's a good point. I'll keep the old js file and just make the ts file under different name. Good catch.

Comment (jtakalai) [reply test/integration/config.ts:1]: seems IDEA refactors don't spot that kind of problems  :x  worrying.

**Comment by jtakalai** [comment]
Some unit tests all passed but the test itself still failed "Error: The action has timed out." 

Add dataunion tests pass, too, so I'm calling this a success.

**Pull request merged**